third_party/wrt/BUILD.gn

```diff
@@ -0,0 +1,90 @@
+import("//build/buildflag_header.gni")
+import("//mojo/public/tools/bindings/mojom.gni")
+import("//services/service_manager/public/service_manifest.gni")
+import("//tools/grit/grit_rule.gni")
+
+# Meta target that includes both wrt_lib and ewk/efl_integration:chromium-ewk.
+group("wrt") {
+  public_deps = [
+    ":wrt_lib",
+    "ewk/efl_integration:chromium-ewk",
+  ]
+}
+
+wrt_lib_sources = [
+  "common/arraysize.h",
+  "common/string_utils.cc",
+  "common/string_utils.h",
+  "ewk/efl_integration/wrt/dynamicplugin.cc",
+  "ewk/efl_integration/wrt/dynamicplugin.h",
+  "ewk/efl_integration/wrt/v8widget.cc",
+  "ewk/efl_integration/wrt/v8widget.h",
+  "ewk/efl_integration/wrt/wrt_dynamicplugin.cc",
+  "ewk/efl_integration/wrt/wrt_dynamicplugin.h",
+  "ewk/efl_integration/wrt/wrtwidget.cc",
+  "ewk/efl_integration/wrt/wrtwidget.h",
+  "xwalk_extensions/common/message_generator_wrt.cc",
+  "xwalk_extensions/common/message_generator_wrt.h",
+  "xwalk_extensions/common/xwalk_extension.cc",
+  "xwalk_extensions/common/xwalk_extension.h",
+  "xwalk_extensions/common/xwalk_extension_adapter.cc",
+  "xwalk_extensions/common/xwalk_extension_adapter.h",
+  "xwalk_extensions/common/xwalk_extension_constants.cc",
+  "xwalk_extensions/common/xwalk_extension_constants.h",
+  "xwalk_extensions/common/xwalk_extension_instance.cc",
+  "xwalk_extensions/common/xwalk_extension_instance.h",
+  "xwalk_extensions/common/xwalk_extension_manager.cc",
+  "xwalk_extensions/common/xwalk_extension_manager.h",
+  "xwalk_extensions/common/xwalk_extension_messages.h",
+  "xwalk_extensions/common/xwalk_extension_server.cc",
+  "xwalk_extensions/common/xwalk_extension_server.h",
+  "xwalk_extensions/renderer/xwalk_extension_client.cc",
+  "xwalk_extensions/renderer/xwalk_extension_client.h",
+  "xwalk_extensions/renderer/xwalk_extension_module.cc",
+  "xwalk_extensions/renderer/xwalk_extension_module.h",
+  "xwalk_extensions/renderer/xwalk_extension_renderer_controller.cc",
+  "xwalk_extensions/renderer/xwalk_extension_renderer_controller.h",
+  "xwalk_extensions/renderer/xwalk_module_system.cc",
+  "xwalk_extensions/renderer/xwalk_module_system.h",
+  "xwalk_extensions/renderer/xwalk_v8tools_module.cc",
+  "xwalk_extensions/renderer/xwalk_v8tools_module.h",
+  "xwalk_extensions/renderer/widget_module.cc",
+  "xwalk_extensions/renderer/widget_module.h",
+  "xwalk_extensions/renderer/object_tools_module.cc",
+  "xwalk_extensions/renderer/object_tools_module.h",
+  "xwalk_extensions/renderer/runtime_ipc_client.cc",
+  "xwalk_extensions/renderer/runtime_ipc_client.h",
+]
+
+static_library("wrt_lib") {
```

Comment:
component target is more flexible than static_library.
How about changing GN target to component?

Reply:
chrome/renderer/BUILD.gn has dependency on wrt as below.
if (enable_castanets) {
    deps += [ "//third_party/wrt" ]
}
And chrome gn target is static, so wrt gn target should be static.

Reply:
I merged chromium-ewk gn target and wrt gn target to "wrt" gn target.

Reply:
I applied all the comments Insoon has left in the previous patchset.
Could you please review this change?